app/models/account_request.rb

```diff
@@ -1,2 +0,0 @@
-class AccountRequest < ApplicationRecord
```

Comment:
Do we want to delete this table at some future date? Or leave the records around? @dfe-teaching-vacancies

Reply:
Hmm we didn't do anything with it besides passing it into a mailer. I know the table was included in analytics.yml, but I'm not sure what's done with this file to be honest.

Reply:
I think killing it is fine - don't know why this existed in the first place given all that happens is it being sent somewhere 👍